libraries/lib-audio-devices/AudioIOBase.cpp

```diff
@@ -27,11 +27,11 @@ Paul Licameli split from AudioIO.cpp
 #include "portmixer.h"
 #endif
 
-int AudioIOBase::mCachedPlaybackIndex = -1;
-std::vector<long> AudioIOBase::mCachedPlaybackRates;
-int AudioIOBase::mCachedCaptureIndex = -1;
-std::vector<long> AudioIOBase::mCachedCaptureRates;
-std::vector<long> AudioIOBase::mCachedSampleRates;
+std::map<int, std::vector<long>> AudioIOBase::mCachedPlaybackRates;
+std::map<int, std::vector<long>> AudioIOBase::mCachedCaptureRates;
+std::map<std::pair<int, int>, std::vector<long>> AudioIOBase::mCachedSampleRates;
+int AudioIOBase::mCurrentPlaybackIndex;
+int AudioIOBase::mCurrentCaptureIndex;
```

Comment:
please initialize with the default values

Reply:
Thanks!